snippets/card-product.liquid

```diff
@@ -152,7 +152,7 @@
                     type="submit"
                     name="add"
                     class="quick-add__submit button button--full-width button--secondary"
-                    aria-haspopup="dialog"
+                    aria-describedby="a11y-modal-window-message"
```

Comment:
This one originally meant that the success message ("cart notification") would apear as a modal.
Is that use correct?

If so, should we have the same attribute on the product page's `Add to Cart`?

Reply:
Yes, and again, only when JS is available.